base/variables.scss

```diff
@@ -10,13 +10,15 @@ $header-font-family: $gibson;
 // Font Sizes
 $base-font-size: 18px;
 $quote-font-size: $base-font-size * 0.9;
-$h1-font-size: 3.25em;
-$h2-font-size: 2.5em;
-$h3-font-size: 1.75em;
-$h4-font-size: 1.2em;
+$h1-font-size: 2.5em;
+$h1-dark-font-size: 1.875em;
+$h2-font-size: 1.5em;
+$h3-font-size: 1em;
+$h4-font-size: 1.125em;
```

Comment:
Then modify the `h4` local to that component, or – better yet – use a `span` with a specific class for that situation instead.